wallets/server/protocols/clink.js

```diff
@@ -0,0 +1,47 @@
+import { WALLET_CREATE_INVOICE_TIMEOUT_MS } from '@/lib/constants'
+import { msatsToSats } from '@/lib/format'
+import { decodeBech32, generateSecretKey, SendNofferRequest, SimplePool } from '@shocknet/clink-sdk'
+
+export const name = 'CLINK'
+
+// https://clinkme.dev/specs.html
+const ERR_INVALID_AMOUNT = 5
+
+export async function createInvoice (
+  { msats, description, expiry },
+  { noffer },
+  { signal }) {
+  const { data: { offer, relay, pubkey } } = decodeBech32(noffer)
+
+  const pool = new SimplePool()
```

Comment:
This is leaking connections.

We're creating a new pool for every invoice, which means we're creating a new websocket connection for each invoice and that connection isn't closed after we received the response. Not sure when it is.

This might block this PR by #2073, same issue for #2531